run-locally/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+  postgres:
+    image: "postgres:12-alpine"
+    environment:
+      - POSTGRES_USER=folio
+      - POSTGRES_PASSWORD=folio
+      - POSTGRES_DB=folio
+    ports:
+      - "5432:5432"
+  zookeeper:
+    image: wurstmeister/zookeeper
+    container_name: zookeeper
+    restart: always
+    ports:
+      - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka
+    container_name: kafka
+    restart: always
+    ports:
+      - "9092:9092"
+      - "29092:29092"
+    environment:
+      KAFKA_LISTENERS: INTERNAL://:9092,LOCAL://:29092
+      KAFKA_ADVERTISED_LISTENERS: INTERNAL://:9092,LOCAL://localhost:29092
```

Comment:
@marc-johnson 
The entry `LOCAL://localhost:29092` in KAFKA_ADVERTISED_LISTENERS is needed to make the jar (running without being wrapped in a Docker container) to work.

Can I replace `LOCAL://:29092` by `LOCAL://localhost:29092` in KAFKA_ADVERTISED_LISTENERS in https://github.com/folio-org/folio-tools/blob/master/infrastructure/local/docker-compose.yml to avoid yet another docker-compose.yml?

Currently the port 29092 duplicates 9092 in infrastructure/local/docker-compose.yml and therefore has no use.

Reply:
Yes, I'm totally happy with that change. I'm glad you suggested it. Thank you.

When I put shared docker compose file together I was rushing to get schema upgrade testing done and did not (and still don't) fully understand Kafka configuration. Please do make improvements to it.

If you would also like to upgrade PostgreSQL to version 12 as well, that would be great too.